tools/make_hgrid/create_gnomonic_cubic_grid.c

```diff
@@ -166,33 +168,36 @@ void create_gnomonic_cubic_grid( char* grid_type, int *nlon, int *nlat, double *
 
   for (nn=0; nn < MAX_NESTS; nn++) {
     ni_nest[nn] = 0;
-    nj_nest[nn] = 0;  
+    nj_nest[nn] = 0;
     ni_parent[nn] = 0;
-    nj_parent[nn] = 0;  
+    nj_parent[nn] = 0;
   }
 
   ntiles2=ntiles;
   global_nest=0;
 
-  if(num_nest_grids && parent_tile[0]== 0)
+  if(num_nest_grids && parent_tile[0]== 0){
     global_nest = 1;
+    //TODO: IMA related. Nncrease ntiles2 by 1 and will decrease some IMA below, but will also change to
+    //  0 two angle_dx and angle_dy values from GR pre multi-nest on tile1.nc
+  }
```

Comment:
Was this TODO intended for longer-term or for this issue? I think technical comments like this are very helpful, just wanted to check. Maybe define IMA?

Reply:
(Whoops, I see you defined IMA a few lines below!)